src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -0,0 +1,48 @@
+package org.broadinstitute.dsde.workbench.leonardo.service
+
+import akka.http.scaladsl.model.StatusCodes
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.workbench.leonardo.dao.DataprocDAO
+import org.broadinstitute.dsde.workbench.leonardo.model.{Cluster, ClusterRequest, ClusterResponse, ClusterStatus, LeoException}
+import org.broadinstitute.dsde.workbench.leonardo.db.{DataAccess, DbReference}
+import org.broadinstitute.dsde.workbench.leonardo.model.ModelTypes.GoogleProject
+import slick.dbio.DBIO
+
+import scala.concurrent.{ExecutionContext, Future}
+
+case class ClusterNotFoundException(googleProject: GoogleProject, clusterName: String) extends LeoException(s"Cluster $googleProject/$clusterName not found", StatusCodes.NotFound)
+
+class LeonardoService(gdDAO: DataprocDAO, dbRef: DbReference)(implicit val executionContext: ExecutionContext) extends LazyLogging {
```

Comment:
our class names are inconsistent. we should decide whether things are called `LeoFoo` or `LeonardoFoo`. i vote `LeoFoo` but we keep the package and repo named `leonardo`